proposals/eligibility_validator_test.go

```diff
@@ -293,48 +296,6 @@ func TestCheckEligibility_KeyMismatch(t *testing.T) {
 	require.False(t, eligible)
 }
 
-func TestCheckEligibility_ZeroTotalWeight(t *testing.T) {
```

Comment:
A node cannot have 0 weight an ATX with `NumUnits` 0 is not accepted as a valid ATX.